recipes/openblas/all/conanfile.py

```diff
@@ -64,100 +78,108 @@ def validate(self):
         if hasattr(self, "settings_build") and cross_building(self, skip_x64_x86=True):
             raise ConanInvalidConfiguration("Cross-building not implemented")
 
-    def source(self):
-        get(self,
-            **self.conan_data["sources"][self.version],
-            strip_root=True,
-            destination=self.source_folder
-        )
+        if self.options.build_relapack:
+            if not self.options.build_lapack:
+                raise ConanInvalidConfiguration("build_relapack option requires build_lapack")
```

Comment:
```suggestion
                raise ConanInvalidConfiguration(f'"{self.name}/*:build_relapack=True" option requires "{self.name}/*:build_lapack=True"')
```

After being bitten a few times by unclear messages in the validate message, from now I'd appreciate these kind of messages that ask the user to set a specific option to be abkle to copy&paste directly form the message